Formula/azure-cli@2.0.rb

```diff
@@ -0,0 +1,54 @@
+require "tmpdir"
+
+class AzureCliAT20 < Formula
+  include Language::Python::Virtualenv
+
+  desc "Microsoft Azure CLI 2.0"
+  homepage "https://docs.microsoft.com/en-us/cli/azure/overview"
+  url "https://azurecliprod.blob.core.windows.net/releases/azure-cli_packaged_0.2.9.tar.gz"
+  sha256 "6f949162d6bbf6848177cc009d60520c12cf2c1dfc068e9e5bbeac0ff6e55e4f"
+  head "https://github.com/Azure/azure-cli.git"
+
+  depends_on :python
+
+  def install
+    virtualenv_create(libexec)
+    bin_dir = libexec/"bin"
+
+    # Install CLI components and dependencies
+    Dir.mktmpdir do |tmp_dir|
```

Comment:
Why can this not be done in the existing temporary directory Homebrew already runs `install` in?